test/modules/toolbox/i18n/toolbox.provider.test.js

```diff
@@ -41,10 +46,17 @@ describe('i18n for menu module', () => {
 		expect(map.toolbox_measureTool_stats_length).toBe('Length');
 		expect(map.toolbox_measureTool_stats_area).toBe('Area');
 		expect(map.toolbox_measureTool_start_new).toBe('Start New');
+		expect(map.toolbox_measureTool_delete_point).toBe('Delete last point');
+		expect(map.toolbox_measureTool_delete_measure).toBe('Delete measure');
+		expect(map.toolbox_measureTool_measure_active).toBe('Tap in the map to start measurement.');
+		expect(map.toolbox_measureTool_measure_draw).toBe('Tap in the map to continue drawing the line (double-tap to finish).');
+		expect(map.toolbox_measureTool_measure_modify).toBe('Tap on the measurement, to add point;</br> Tap then drag to move the point;</br> Single tap on the point to delete.');
```

Comment:
To add a point, tap on a measurement;</br>To move a point, press and drag it;</br>To delete a point, tap on it